ui/app/components/app/gas-customization/gas-modal-page-container/index.scss

```diff
@@ -56,21 +56,23 @@
       display: none;
     }
 
+    &__tab {
+      margin-right: 0;
+    }
+
     &__tabs {
       margin-top: 0;
     }
 
     &__tab {
       width: 100%;
-      font-size: 14px;
 
-      &:last-of-type {
-        margin-right: 0;
+      & button {
+        font-size: 14px;
       }
 
-      &--selected {
+      &.tab--active button {
         color: $primary-blue;
-        border-bottom: 2px solid $primary-blue;
```

Comment:
question: Seems like the border-bottom got removed? was this intentional? Was it made unnecessary by using a button?

Reply:
Yep!

<img width="455" alt="Border" src="https://user-images.githubusercontent.com/46655/96744733-f9a9c700-138a-11eb-8cb5-db2eae583bd4.png">